deployment/terraform/live/main.tf

```diff
@@ -9,18 +9,35 @@ terraform {
 
 provider "aws" {
   version = "~> 1.0"
-
-  region = "${var.region}"
+  region  = "${var.region}"
 }
 
 module "object_storage" {
   source           = "../modules/s3"
-  bucket_name_base = "${terraform.workspace}"
+  bucket_name_base = "${replace(terraform.workspace, "/[^A-Za-z0-9]/", "-")}"
 }
 
 module "identity_pool" {
   source                   = "../modules/cognito"
-  identity_pool_name       = "${var.identity_pool_name}"
+  name                     = "${terraform.workspace} identity pool"
+  identity_pool_name       = "${replace(terraform.workspace, "/[^A-Za-z0-9]/", "")}"
   upload_bucket_name       = "${module.object_storage.upload_bucket_name}"
   iam_resource_name_prefix = "${terraform.workspace}"
 }
+
+module "vpc" {
+  source               = "../modules/vpc"
+  name                 = "${terraform.workspace} vpc"
```

Comment:
Name parameter is only needed when creating multiple instances of a module which does not appear to be the case for this module https://www.terraform.io/docs/modules/usage.html#multiple-instances-of-a-module

Reply:
pushed fix